rasa/core/channels/hangouts.py

```diff
@@ -319,13 +321,10 @@ async def receive(request: Request) -> HTTPResponse:
                     )
                 )
             except CancelledError:
-                logger.error(
-                    "Message handling timed out for " "user message '{}'.".format(text)
-                )
+                structlogger.error("hangouts.message.blueprint", text=text)
             except Exception as e:
-                logger.exception(
-                    f"An exception occurred while handling user message: {e}, "
-                    f"text: {text}"
+                structlogger.exception(
+                    "hangouts.message.blueprint", text=text, exception=e
```

Comment:
```suggestion
                    "hangouts.message.blueprint.failure", text=text, exception=e
```

Reply:
exception is not needed, `structlogger.exception(...)` automatically captures the exception